src/main/kotlin/com/hypto/iam/server/authProviders/MicrosoftAuthProvider.kt

```diff
@@ -47,12 +47,16 @@ object MicrosoftAuthProvider : BaseAuthProvider, KoinComponent {
             microsoftUser.mail,
             microsoftUser.displayName,
             "",
-            getProviderName()
+            getProviderName(),
+            mapOf(
+                "id" to microsoftUser.id,
```

Comment:
is `microsoftUser.id == oid` ? Can we add a comment linking to microsoft docs to our code to help future developers understand easily

Reply:
Will add the [blog](https://0x8.in/blog/2021/04/30/mip-oid-sub/) link. Will that suffice?